noetic/distribution.yaml

```diff
@@ -1113,19 +1113,35 @@ repositories:
     doc:
       type: git
       url: https://github.com/4am-robotics/cob_control.git
-      version: melodic_release_candidate
+      version: noetic-release-candidate
     release:
       packages:
-      - cob_base_controller_utils
       - cob_base_velocity_smoother
-      - cob_cartesian_controller
       - cob_collision_velocity_filter
       - cob_control
+      - cob_footprint_observer
+      - cob_hardware_emulation
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/4am-robotics/cob_control-release.git
+      version: 0.8.24-1
+    source:
+      type: git
+      url: https://github.com/4am-robotics/cob_control.git
+      version: noetic-devel
+    status: maintained
+  cob_control_legacy:
```

Comment:
similar to:
- https://github.com/ros/rosdistro/pull/40670/files#r1570307215
- https://github.com/ros/rosdistro/pull/40671/files#r1570462148

I hope I'm doing the right thing here - could you please confirm that this PR will:
- release a new version `0.8.24` of the following packages from `noetic-release-candidate`
      - cob_base_velocity_smoother
      - cob_collision_velocity_filter
      - cob_control
      - cob_footprint_observer
      - cob_hardware_emulation
- keep the last release `0.8.23` for the following packages available based on `indigo_release_candidate`
      - cob_base_controller_utils
      - cob_cartesian_controller
      - cob_control_mode_adapter
      - cob_control_msgs
      - cob_frame_tracker
      - cob_mecanum_controller
      - cob_model_identifier
      - cob_obstacle_distance
      - cob_omni_drive_controller
      - cob_trajectory_controller
      - cob_tricycle_controller
      - cob_twist_controller